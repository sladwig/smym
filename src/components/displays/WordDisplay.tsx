import React from 'react';
import { display } from './display';
import { WordT, EmptyWordT } from './types';

export const WordDisplay = ({ word }: { word: WordT | EmptyWordT }) => {
    return <>{word.characters.map(display)}</>;
};
