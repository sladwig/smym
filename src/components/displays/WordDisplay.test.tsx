import { render } from '@testing-library/react';
import React from 'react';
import { WordDisplay } from './WordDisplay';

describe('WordDisplay', () => {
    test('renders', () => {
        const { container } = render(<WordDisplay />);
        expect(container).toMatchSnapshot();
    });
});
