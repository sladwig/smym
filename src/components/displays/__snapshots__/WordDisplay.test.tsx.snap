// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordDisplay renders 1`] = `<div />`;
